chirp/drivers/kg935g.py

```diff
@@ -82,14 +83,17 @@
 VOX_LIST = ["Off"] + ["%s" % x for x in range(1, 10)]
 BANDWIDTH_LIST = ["Narrow", "Wide"]
 VOICE_LIST = ["Off", "On"]
-LANGUAGE_LIST = ["Chinese", "English"]
+LANGUAGE_LIST = ["CN", "English", "TC"]
```

Comment:
Yeah... I was trying to match what the radio shows for the settings to be consistent, but I understand the point about clarity.  I will make the changes.   Thanks for giving me clarity on how you would this to be.  I bounce back and forth between maintaining consistency with the radio and clarity.  Clarity will win going forward.

Reply:
Yeah, I wish the radio was more consistent and that we could mirror it. However, the radio is also more limited in what it can display, so if it's abbreviating (even if inconsistently) I think it's fine (and preferable) to take the  more readable option. If using a computer to program it wasn't easier, we wouldn't need CHIRP :)